sessions/007/README.md

```diff
@@ -173,9 +174,11 @@ optimizationResult:
 sessionId: f0605d40-37be-43b9-be1f-83167633f37c
 ```
 
-When the proposal is ready, we can approve it by using another annotation and wait for CC to move partitions ande replica roles around.
+When the proposal is ready, we can approve it by using another annotation and wait for Cruise Control to move partitions and replica roles around.
 When the rebalance is ready, we can look at the result and see if the new broker has picked some of the existing partitions.
 
+> To enable auto-approval, you add `strimzi.io/rebalance-auto-approval: "true"` as an annotation to the `KafkaRebalance` custom resource before you generate the proposal.
```

Comment:
This course is using AMQ Streams 2.2.0 (LTS), so this most wanted feature is not available. Can you please remove it or rephrase saying it's available upgrading to 2.3.0?